README.md

```diff
@@ -1,3 +1,30 @@
-[![Build Status](https://travis-ci.org/kpi-web-guild/django-girls-blog-PetrushynskyiOleksii.svg?branch=master)](https://travis-ci.org/kpi-web-guild/django-girls-blog-PetrushynskyiOleksii)
+# Django Girls blog Petrushynskyi [![Build Status](https://travis-ci.org/kpi-web-guild/django-girls-blog-PetrushynskyiOleksii.svg?branch=master)](https://travis-ci.org/kpi-web-guild/django-girls-blog-PetrushynskyiOleksii)
 
-# Django-girls-blog-PetrushynskyiOleksii
+### Installation
+- [Locally](https://github.com/kpi-web-guild/django-girls-blog-PetrushynskyiOleksii#Lokaly)
```

Comment:
Actually you could omit URL part and make the target relative (as easy as `#Lokaly`)

Reply:
Oh, btw it doesn't work now because of a typo